### PR TITLE
MongoDB integration

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20026

## Minimum version of Demisto
- [x] 4.5.0
## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

Documentation:
https://github.com/demisto/content/blob/mongodb-final/Integrations/MongoDB/README.md

### CODE REVIEW

File: Packs/MongoDB/Integrations/MongoDB/MongoDB.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+        self,
+        urls: List[str],
+        username: str,
+        password: str,
+        database: str,
+        ssl: bool = False,
+        insecure: bool = False
+    ):
+        if insecure and not ssl:
+            return_error(f'"Trust any certificate (not secure)" must be ticked with "Use TLS/SSL secured connection"')
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDB/MongoDB.py]: why is this restriction? where is that documented on pymongo?

File: Packs/MongoDB/Integrations/MongoDB/MongoDB.py [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    _ = client.create_collection(collection)
+    return (
+        f'MongoDB: Collection \'{collection}\' has been successfully created.',
+        None,
+    )
+
+
+def list_collections_command(client: Client, **kwargs) -> Tuple[str, dict, list]:
+    raw_response = client.db.list_collection_names()
+    if len(raw_response):
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDB/MongoDB.py]:
```suggestion
    if raw_response:
```

File: Packs/MongoDB/Integrations/MongoDB/MongoDB.py [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+            'MongoDB: All collections in database', raw_response, headers=['Collection']
+        )
+        outputs = {
+            'MongoDB.Collection(val.Name === obj.Name)': [
+                {'Name': collection} for collection in raw_response
+            ]
+        }
+        return readable_outputs, outputs, raw_response
+    else:
+        return "MongoDB: No results found", {}, raw_response
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDB/MongoDB.py]:
```suggestion
        return 'MongoDB: No results found', {}, raw_response
```

File: Packs/MongoDB/Integrations/MongoDB/MongoDB.py (hunk 1 above)
Comment (jochman) [reply Packs/MongoDB/Integrations/MongoDB/MongoDB.py]: when you're using them both, that's the error raised.

Comment (Itay4) [reply Packs/MongoDB/Integrations/MongoDB/MongoDB.py]: what error?

Comment (jochman) [reply Packs/MongoDB/Integrations/MongoDB/MongoDB.py]:
```
ConfigurationError: ssl has not been enabled but the following ssl parameters have been set: ssl_cert_reqs. Please set `ssl=True` or remove.
```

**Pull request merged**